src/main/java/emu/grasscutter/game/world/Scene.java

```diff
@@ -75,6 +76,12 @@ public Scene(World world, SceneData sceneData) {
         this.sceneData = sceneData;
         this.players = new CopyOnWriteArrayList<>();
         this.entities = new ConcurrentHashMap<>();
+        this.removedEntities = new LinkedHashMap<>(){
```

Comment:
Please, try not using this types of hacks, it's obvious that this is wrong, so don't use it, try to figure what is the actual problem and not using and old removed code

Reply:
Sorry. I checked hartie's and I checked your main branches and saw the code still there, so I figured the code was accidentally deleted and needed to be reinstated.